.github/ISSUE_TEMPLATE/planned--specific-support-new-oss.md.md

```diff
@@ -0,0 +1,87 @@
+---
+name: 'Planned: Specific support new OSs'
+about: Test compatibility with new OS.
+title: Support new OSs - <OS name & version> - <Specific test name>
+labels: level/task, request/operational, type/maintenance
+assignees: ''
+
+---
+
+# Description
+| Related issue | Epic issue |
+|---|---|
+| Issue number | Issue number|
+
+| Agent tier | Central components support | OS type |
+|-|-|-|
+| 1/2/3 | Yes/No | New family/Major/Minor |
+
+# Plan
+
+<!-- Uncomment for QA issue
+**QA**
+- [ ] **Major/New family**: Add new tests according to the OS's tier.
+- [ ] **Major/New family**: Add proper documentation.
+-->
+
+<!-- Uncomment for DEVOPS issue
+**DevOps**
+- [ ] **Major/New family**: Add support in the allocator. This is the first step for everything else, regardless of the team as we need to be able to allocate OS VMs to generate a testing environment.
+- [ ] **Minor**: Update the allocator images.
+- [ ] **Central components**: Review/test the installation assistant using the new OS.
+- [ ] Update AMI, OVA, or Docker images if needed.
+-->
+
+<!-- Uncomment for CPPSERVER issue
+**CppServer**
+- [ ] **Central components**: Add support for the new OS to the GitHub Actions package builder.
+- [ ] **Central components**: Smoke test that the package works, including installation and upgrade.
```

Comment:
We always rely on our internal documentation to update the template as well:
```suggestion
- [ ] **Central components**: Smoke test that the package works, including installation and upgrade.
- [ ] **Major/New family**: Make sure that VD works properly according to the OS tier.
```

Reply:
done